CONTRIBUTING.txt

```diff
@@ -0,0 +1,99 @@
+    7. A PR SHALL NOT include non-trivial code from other projects unless the Contributor is the original author of that code.
+    8. A PR MUST compile cleanly and pass project self-tests on all target Platforms.
+    9. Each commit message in a PR MUST succinctly explain what the commit achieves.
+    10. The PR Description MUST consist of a single short (less than 50 characters) line stating the problem (“Problem: …") being solved, followed by a blank line and then the proposed solution (“Solution: …").
+    11. A “Valid PR” is one which satisfies the above requirements.
+
+6. Process
+
+    1. Change on the project follows the pattern of accurately identifying problems and applying minimal, accurate solutions to these problems.
+    2. To request changes, a User (end user, Contributor, Maintainer, or Administrator) logs an issue on the project Platform issue tracker.
+    3. The User or Contributor SHOULD write the issue by describing the problem they face or observe. Links to the forum or other resources are permitted but the issue SHOULD be complete and accurate and SHOULD NOT require the reader to visit the forum or any other Platform to understand what is being described.
+    4. Issue authors SHOULD strive to describe the minimum acceptable condition.
+    5. Issue authors SHOULD focus on User tasks and avoid comparisons to other software solutions.
+    6. The User or Contributor SHOULD seek consensus on the accuracy of their observation and the value of solving the problem.
+    7. To submit a solution to a problem, a Contributor SHALL create a pull request back to the project.
+    8. Contributors and Maintainers SHALL NOT commit changes directly to the target branch.
+    9. To discuss a proposed solution, Users MAY comment on the Platform Pull Request. Forum conversations regarding the solution SHOULD be discouraged and conversation redirected to the Pull Request or the related issue.
+    10. To accept or reject a Pull Request, a Maintainer SHALL use the Platform interface.
+    11. Maintainers SHOULD NOT merge their own PRs except:
+      - in exceptional cases, such as non-responsiveness from other Maintainers for an extended period (more than 1-2 days).
+      - If the Maintainer is also the Primary Developer of the workbench or subsystem.
+
+    12. Maintainers SHALL merge valid PRs from other Contributors rapidly.
+    13. Maintainers MAY, at their discretion merge PRs that have not met all criteria to be considered valid to:
+        - (a) end fruitless discussions
+        - (b) capture toxic patches in the historical record
+        - (c) engage with the Contributor on improving their patch quality.
+    14. Maintainers SHALL NOT make value judgments on correct patches.
+    15. Any Contributor who has value judgments on a PR SHOULD express these via their own PR.
+    16. The User who created an issue SHOULD close the issue after checking the PR is successful.
```

Comment:
What defines "the PR is successful" ? Merged ? Tested ?
Also be aware that this contradicts current process. ;)

Reply:
This is wishful thinking and not realistic

Reply:
@Zolko-123 please try to stay away from destructive attitude. What's you're counter proposal so issues are closed when fixed?

Reply:
Maintainers should check Issues and close them when 1) inactive or 2) wrong or 3) completed

Reply:
16. The originator of an issue SHOULD close the issue after verifying that the merged PR solves the problem stated in the PR or original issue.

Reply:
Resolving due to lack of activity.  The document has changed around this area and seems clearer now.